airflow/contrib/utils/gcp_field_validator.py

```diff
@@ -210,7 +209,17 @@ def _get_field_name_with_parent(field_name, parent):
     @staticmethod
     def _sanity_checks(children_validation_specs, field_type, full_field_path,
                        regexp, allow_empty, custom_validation, value):
-        # type: (dict, str, str, str, Callable, object) -> None
```

Comment:
This needs to stay: This comment is very specific and used for Type checking hinting -- doc comment's aren't picked up by any tools https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code

Reply:
There are *zero* docs for it, but https://github.com/theacodes/sphinx-docstring-typing might help here.

Reply:
I think we have changed the purpose of this PR. I will create a new ticket in Jira about the problems we encountered. In this PR, I will undo the changes that apply to Python 3 type checking hinting. They should not be here. Do you agree with me?